unimatrix.py

```diff
@@ -60,6 +60,8 @@
                        refreshing, 100 uses none. Use negative numbers for
                        even lower speeds. Default=85
 
+  -S SPACE             Integer up to 5. space between coloumns 
```

Comment:
```suggestion
  -S SPACE             Integer up to 5. Space between columns
```
@prownd Any chance you're still around? I've been inactive for a long time, but I'm picking up the project again. If you respond I'll make some comments about this PR. Otherwise, I'll try to include something similar in v2 in [MTRX-2](https://trello.com/c/OsiLAOkC/1-mtrx-2-move-character-sets-to-configuration-files)

Reply:
I'm sorry, for some reason, I haven't been active for a long time 
Yes, your suggestions for modification are correct. 
There are indeed some grammar and spelling errors